SocialFilter/sections/specific.txt

```diff
@@ -57,6 +57,7 @@ univ-study.net,sittaka-information.com,math-fun.net,polaris-official.com,apexhea
 !
+upinews.kr##.viewSns
```

Comment:
On desktop it also hides buttons to change font size.
Maybe something like this:
```adblock
m.upinews.kr##.viewSns
upinews.kr##.viewSns > ul:not(.font)
```
will be okay?

Reply:
No. I think that the feature is being provided for a visitor who has bad sight.
So, It should not be blocked by any social filter.

Reply:
@piquark6046 I mean that this rule:
```adblock
upinews.kr##.viewSns
```
causes that these icons are hidden.

I mean these icons:
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/200016179-1a5fc55d-ab83-416d-aac6-0d8b2843d2fe.png)

</details>

That how it looks like if I add `upinews.kr##.viewSns` to user rules:
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/200016466-5f6947c0-7b47-4aa0-b9ec-393b4d5d8635.png)

</details>

So, I proposed a different rules which should not hide those buttons:
```adblock
upinews.kr##.viewSns > ul:not(.font)
upinews.kr###viewWrap > .viewSns
```
Second rule is needed for mobile view.

Reply:
@AdamWr

Reply:
Thank you.